src/ExperimentTable.js

```diff
@@ -114,14 +114,13 @@ class ExperimentTable extends React.Component {
 
       this.sort(aaData).filter(data => data &&
         Object.keys(data).map(key => displayedFields.includes(key) ? data[key] : null)
-          .some(value => value && value.toString().includes(selectedSearch))) :
+          .some(value => value && value.toString().toLocaleLowerCase().includes(selectedSearch))) :
```

Comment:
Is there a particular reason why you use `toLowercase` for `state.selectedSearch`, and `toLocaleLowerCase` to filter the data?